.develop/Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.20
```

Comment:
do we use 1.21? I think in vars we have 1.20

Reply:
go version only pinned in dockerfiles/go.mod so I just yolo'd since you had 1.21 on the clabverter thing :D